Source/Initialization/PlasmaInjector.cpp

```diff
@@ -639,6 +639,22 @@ void PlasmaInjector::parseMomentum (amrex::ParmParse& pp)
         h_inj_mom.reset(new InjectorMomentum((InjectorMomentumGaussianFlux*)nullptr,
                                              ux_m, uy_m, uz_m, ux_th, uy_th, uz_th,
                                              flux_normal_axis, flux_direction));
+    } else if (mom_dist_s == "uniform") {
+        amrex::Real ux_min = 0._rt;
+        amrex::Real uy_min = 0._rt;
+        amrex::Real uz_min = 0._rt;
+        amrex::Real ux_max = 0._rt;
+        amrex::Real uy_max = 0._rt;
+        amrex::Real uz_max = 0._rt;
+        utils::parser::queryWithParser(pp, "ux_min", ux_min);
+        utils::parser::queryWithParser(pp, "uy_min", uy_min);
+        utils::parser::queryWithParser(pp, "uz_min", uz_min);
+        utils::parser::queryWithParser(pp, "ux_max", ux_max);
+        utils::parser::queryWithParser(pp, "uy_max", uy_max);
+        utils::parser::queryWithParser(pp, "uz_max", uz_max);
+        // Construct InjectorMomentum with InjectorMomentumUniform.
+        h_inj_mom.reset(new InjectorMomentum((InjectorMomentumUniform*)nullptr,
```

Comment:
@ax3l Should we check before this line that for all directions `min` is never larger than `max`?

Reply:
The distribution is symmetrical and works either way. If `min` is larger than `max`, the momentum is still going to be generated between `min` and `max` (or `max` and `min`). It should also work if they're equal, in which case it collapses onto a delta.

Reply:
Ah, missed that 👍 thanks :)